analyzers/rspec/cs/S109_c#.html

```diff
@@ -27,5 +27,12 @@ <h2>Compliant Solution</h2>
 }
 </pre>
 <h2>Exceptions</h2>
-<p>This rule doesn’t raise an issue when the magic number is used as part of the <code>GetHashCode</code> method or a variable/field declaration.</p>
+<p>This rule doesn’t raise an issue when the magic number is used as part of:</p>
+<ul>
+  <li> the <code>GetHashCode</code> method </li>
+  <li> a variable/field declaration </li>
+  <li> the single argument of an attribute </li>
+  <li> a named argument for a method or attribute </li>
+  <li> a constructor call </li>
```

Comment:
I haven't included the single-digit comparison because it's hard to explain and isn't worth the effort TBH